livre/eq/Channel.cpp

```diff
@@ -275,16 +275,9 @@ class Channel
         _renderViewPtr->setViewport( viewport );
 
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
```

Comment:
Manual testing is not feasible imo, so I offer to revert that change as it is not necessary for the feature I want to add. I find it sad though that refactoring code is so hard, but I don't want to add regressions as well.